src/plugins/discover/public/application/components/doc_views/context_app.tsx

```diff
@@ -100,7 +105,7 @@ export function ContextApp({
           onSort={() => {}}
           sort={sort}
           rows={rows}
-          displayTimeColumn={true}
+          displayTimeColumn={displayTimeColumn || true}
```

Comment:
Since `displayTimeColumn` is boolean too, this will be `true` always. I don't think that was your intention.